smithy-trait-codegen/src/main/java/software/amazon/smithy/traitcodegen/generators/FromNodeMapperVisitor.java

```diff
@@ -43,17 +52,29 @@ final class FromNodeMapperVisitor extends ShapeVisitor.DataShapeVisitor<Void> {
     private final TraitCodegenWriter writer;
     private final Model model;
     private final String varName;
+    private final SymbolProvider symbolProvider;
     private final int nestedLevel;
 
     FromNodeMapperVisitor(TraitCodegenWriter writer, Model model, String varName) {
-        this(writer, model, varName, 0);
+        this(writer, model, varName, 0, null);
```

Comment:
If there's a use case in which the `symbolProvider` can be null then we might need to create a different class, if so let's do that instead. Passing a null might lead to issues, so either create a new `FromNodeMapperVisitor` for which `SymbolProvider` is never null, or refactor this one to ensure the same.